htp/htp_response.c

```diff
@@ -1073,6 +1073,7 @@ htp_status_t htp_connp_RES_LINE(htp_connp_t *connp) {
 
                 connp->out_current_consume_offset = connp->out_current_read_offset;
                 htp_status_t rc = htp_tx_res_process_body_data_ex(connp->out_tx, data, len + chomp_result);
+                htp_connp_res_clear_buffer(connp);
```

Comment:
Nice, did you look for other missing calls to `htp_connp_res_clear_buffer` ?

Reply:
The `_FINALIZE()` state, which also calls the response body data hook clears this buffer. I haven't looked everywhere though. This should fix the bug we are experiencing.